### PR TITLE
Add USB 2.0 peripheral for STM32L053 in device tree

### PR DESCRIPTION
**Pull request by gudnimg**
I noticed STM32L053 didn't have any USB FS peripheral set similar to STM32L073. It seems to be a copy-paste from STM32L073 since they are both STM32L0x3.

This pull request also:

* Removes redundant `ram-size` parameter set for STM32F469, it is already set for STM32F405 which is included into STM32F469 `.dtsi` file.
* Add `maximum-speed` parameter for USB peripheral for STM32L072 & STM32L073 and all STM32L4x2, STM32L4x3. I set it to `"full-speed"`. Saw this parameter when looking through STM32F4 `.dtsi` files.

This is my first pull request here, hope this helps. 😄 

image

### CODE REVIEW

File: dts/arm/st/l0/stm32l072.dtsi [hunk 1]
```diff
@@ -19,6 +19,7 @@
 			interrupt-names = "usb";
 			num-bidir-endpoints = <8>;
 			ram-size = <1024>;
+			maximum-speed = "full-speed";
```
Comment (erwango) [inline dts/arm/st/l0/stm32l072.dtsi:22]: This is not strictly required as this property is optional. 
When not available, USB controller set speed to the controller maximum available speed.
This is what is done in STM32 USB driver.
Anyway, this doesn't harm.

**Review by erwango — CHANGES_REQUESTED** [review]
Thanks for this contribution.

The only change requested is to re-compose your pull request. 
-squash all the commits
-split in 2 or 3 commits: 1 commit to introduce the new node, other commit(s) for the L4/F4 fixes.
For each commit, please provide a commit message in accordance with contributing guidelines (you can also check other commit messages)

**Comment by GunZi200** [comment]
> Thanks for this contribution.
> 
> The only change requested is to re-compose your pull request.
> -squash all the commits
> -split in 2 or 3 commits: 1 commit to introduce the new node, other commit(s) for the L4/F4 fixes.
> For each commit, please provide a commit message in accordance with contributing guidelines (you can also check other commit messages)

Do I understand it correctly that this would require a new branch / fork to change the previous commit messages?

**Comment by erwango** [comment]
> > Thanks for this contribution.
> > The only change requested is to re-compose your pull request.
> > -squash all the commits
> > -split in 2 or 3 commits: 1 commit to introduce the new node, other commit(s) for the L4/F4 fixes.
> > For each commit, please provide a commit message in accordance with contributing guidelines (you can also check other commit messages)
> 
> Do I understand it correctly that this would require a new branch / fork to change the previous commit messages?

You don't need a new branch, nor fork to do this. In few steps:
-1)squash all your commits into one
-2)amend the commit, spit the content, rework the commit message, commit
-3)Iterate 2) for the next commits
-4)Force push your branch (git push -f <your_fork> <your_branch_name>)

**Review by erwango — APPROVED** [review]

**Review by FRASTM — COMMENTED** [review]
LGTM including @erwango 's comments

**Review by FRASTM — APPROVED** [review]

**Pull request merged**